app/helpers/application_helper.rb

```diff
@@ -76,4 +76,12 @@ def human_duration(secs)
       "#{n.to_i} #{name}" unless n.to_i.zero?
     end.compact.reverse.join(' ')
   end
+
+  def hide(cond)
+    cond ? ' hide' : ''
+  end
+
+  def show(cond)
+    cond ? '' : ' hide'
+  end
```

Comment:
Helpers to use the foundation classes instead of the inline style `display: none`